src/main/resources/avro/metadatamethods.avdl

```diff
@@ -16,14 +16,14 @@ record SearchDatasetsRequest {
   Specifies the maximum number of results to return in a single page.
   If unspecified, a system default will be used.
   */
-  union { null, int } pageSize = null;
+  int pageSize;
```

Comment:
Again, pageSize shouldn't be required now, but should default to 0 for proto compability. The semantics of pageSize=0 are clear here, it means 'the server decides'.